quil-cli/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "cli"
```

Comment:
Will this allow the package to be `cargo installed` as a binary?

Reply:
Also curious what name this would get installed under. Regardless, if we ever publish this to crates.io, we'll want this to be `quil-cli`, `quil-parser-cli`, or similar

Reply:
I went ahead and named it `quil-cli`, since I could see things like program analysis being added to the CLI.